frontend/src/styles/GlobalStyles.ts

```diff
@@ -0,0 +1,72 @@
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyles = createGlobalStyle`
+
+:root{
+  --strawberry: #f1356d;
+    --lightest: #BFD7ED;
+    --dark: #003B73;
+    --medium: #0074B7;
+    --light: #60A3D9;
+}
+ 
+html {
+    --maxWidth: 1000px;
+    --bs: 0 12px 24px 0 rgba(0,0,0,0,.09)
```

Comment:
All properties starting with `--` are css variables, so these two also should go in the `:root`